.travis.yml

```diff
@@ -1,25 +1,28 @@
 language: go
 
+dist: bionic
```

Comment:
Run on an Ubuntu 18.04 VM. This is the latest Ubuntu currently offered by Travis.